### ISSUE TITLE
Hyperparameters for Experiment2-10k

### ISSUE THREAD

**Issue by Daniel17bn** [Daniel17bn/BSP-RL-Exploration#6]
configs = [
        {
            "name": "Early Exploitation",
            "params": Params(
                total_episodes=10000,
                learning_rate=0.1,
                gamma=0.99,
                epsilon_start=1.0,
                epsilon_decay=0.9985,  # Fast decay - reaches 0.01 around ep 3100
                epsilon_min=0.01,
                seed=42,
                is_rainy=False,
                fickle_passenger=False,
                action_size=6,
                state_size=500,
            )
        },
        {
            "name": "Balanced Exploration",
            "params": Params(
                total_episodes=10000,
                learning_rate=0.1,
                gamma=0.99,
                epsilon_start=1.0,
                epsilon_decay=0.9995,  # Medium decay - reaches 0.01 around ep 9200
                epsilon_min=0.01,
                seed=42,
                is_rainy=False,
                fickle_passenger=False,
                action_size=6,
                state_size=500,
            )
        },
        {
            "name": "Extended Exploration",
            "params": Params(
                total_episodes=10000,
                learning_rate=0.1,
                gamma=0.99,
                epsilon_start=1.0,
                epsilon_decay=0.9998,  # Slow decay - epsilon ~0.82 at ep 10k
                epsilon_min=0.01,
                seed=42,
                is_rainy=False,
                fickle_passenger=False,
                action_size=6,
                state_size=500,
            )
        },
        {
            "name": "High Min Exploration",
            "params": Params(
                total_episodes=10000,
                learning_rate=0.1,
                gamma=0.99,
                epsilon_start=1.0,
                epsilon_decay=0.9995,
                epsilon_min=0.15,  # Never goes below 15% random actions
                seed=42,
                is_rainy=False,
                fickle_passenger=False,
                action_size=6,
                state_size=500,
            )
        },
    ]
